src/js/common/components/questions/FullName.jsx

```diff
@@ -111,6 +115,8 @@ FullName.propTypes = {
       dirty: React.PropTypes.bool,
     }),
   }).isRequired,
+  customValidation: React.PropTypes.func,
```

Comment:
I agree, in general we should rethink how we're handling error messages, it does get confusing.